src/components/Button/src/Button.vue

```diff
@@ -230,9 +238,45 @@ export default {
 	},
 
 	computed: {
+		resolvedSize() {
```

Comment:
Do we want to add validation for resolved computer props to prevent invalid values being passed in via `<m-theme>`?